File: VB/DXPopupMenu_DisplayMode/SimpleForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Utils.Menu
Imports DevExpress.LookAndFeel.Design

Namespace DXPopupMenu_DisplayMode
	Partial Public Class SimpleForm
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Public PopupMenu As DXPopupMenu

		Protected Overrides Sub OnMouseUp(ByVal e As MouseEventArgs)
			If e.Button = MouseButtons.Right Then
				CType(PopupMenu, IDXDropDownControl).Show(New SkinMenuManager(UserLookAndFeelDefault.Default), Me, e.Location)
			End If
		End Sub
	End Class
End Namespace

File: CS/DXPopupMenu_DisplayMode/SimpleForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.Utils.Menu;
using DevExpress.LookAndFeel.Design;

namespace DXPopupMenu_DisplayMode {
    public partial class SimpleForm : Form {
        public SimpleForm() {
            InitializeComponent();
        }

        public DXPopupMenu PopupMenu;

        protected override void OnMouseUp(MouseEventArgs e) {
            if(e.Button == MouseButtons.Right) {
                ((IDXDropDownControl)PopupMenu).Show(new SkinMenuManager(UserLookAndFeelDefault.Default), this, e.Location);
            }
        }
    }
}